internal/play/play.go

```diff
@@ -21,18 +21,29 @@ func NewCommand() *cobra.Command {
 			}
 
 			query := strings.Join(args, " ")
+			queryType := "track"
 
 			deviceID, err := cmd.Flags().GetString("device-id")
 			if err != nil {
 				return err
 			}
 
-			playlistName, err := cmd.Flags().GetString("playlist")
+			contextQuery, err := cmd.Flags().GetString("playlist")
 			if err != nil {
 				return err
 			}
 
-			status, err := Play(api, query, playlistName,  deviceID)
+			if contextQuery == "" {
+				contextQuery, err = cmd.Flags().GetString("album")
+				if err != nil {
+					return err
+				}
```

Comment:
Let's also bring this outside of the `if`. We need to err if a user has passed more than one of:
* track (as args)
* `--album`
* `--playlist`

The error could look like: `Invalid combination of args or flags`.

```suggestion
				album, err := cmd.Flags().GetString("album")
				if err != nil {
					return err
				}
```